src/test_l1_dsVideoDevice.c

```diff
@@ -831,20 +867,26 @@ void test_l1_dsVideoDevice_negative_dsGetSupportedVideoCodingFormats(void)
     DS_ASSERT_AUTO_TERM_NUMERICAL((handle >= 0), true);
 
     // Step 04: Call dsGetSupportedVideoCodingFormats() with an invalid handle
+    #ifdef ENABLE_ENHANCED_ERROR_CODE
```

Comment:
dsERR_INVALID_PARAM is not a ENHANCED_ERROR_CODE
hal has to support

Reply:
done